satellite/peer.go

```diff
@@ -196,10 +191,9 @@ type Peer struct {
 		Inspector *irreparable.Inspector
 	}
 	Audit struct {
-		Service *audit.Service
-		Queue   *audit.Queue
-		Worker  *audit.Worker
-		Chore   *audit.Chore
+		Queue  *audit.Queue
+		Worker *audit.Worker
+		Chore  *audit.Chore
```

Comment:
adding `Reporter` and `Verifier` here could simplify tests

Reply:
okay! they should still be passed into NewWorker too right?